API/app/Http/Controllers/Api/ShoppingListController.php

```diff
@@ -196,10 +201,16 @@ public function getUserShoppingLists(Request $request)
     {
         Log::info("Received shopping-lists request from IP address " . $request->ip());
 
-        $userId = $request->user_id;
+        $user = Auth::user();
+        if ($user) {
+            Log::info("Got user from Auth facade: ".$user);
+        } else {
+            Log::info("COULDN'T GET USER FROM AUTH FACADE");
+        }
+
 
-        Log::info("User ID retrieved from JWT: " . print_r($userId, true));
-        $shoppingLists = ShoppingList::where('user_id', $userId)->get(['list_id', 'name', 'updated_at']); // Retrieve only the necessary fields
+        Log::info("User ID retrieved from sanctum: " . print_r($user->user_id, true));
```

Comment:
Also change this logging to only show on DEBUG mode (add constant to this file if necessary).